packages/core/src/tracing/instrumentation/protocols/httpServer.js

```diff
@@ -203,3 +206,12 @@ function shimEmit(realEmit) {
     });
   };
 }
+function logInstanaHeaders(req) {
+  if (req.headers) {
+    Object.keys(req.headers).forEach(header => {
+      if (header?.toLowerCase()?.startsWith('x-instana')) {
```

Comment:
Can we read that from constants? We can add constants.ISNTANA_HEADER_PREFIX='x-instana'